src/Components/Publishing/Display/__tests__/DisplayAd.test.tsx

```diff
@@ -78,7 +55,16 @@ describe("Display Ad", () => {
   })
 
   it("Tracks display ad impression", () => {
-    expect(trackEvent).toBeCalledWith({
+    const { Component, dispatch } = mockTracking(DisplayAd)
+
+    const component = mount(<Component {...displayAdProps} />)
+
+    component
+      .find(Waypoint)
+      .getElement()
+      .props.onEnter()
```

Comment:
This is a comment on Enzyme, not a comment on this code. The thing I'd rather test here is "when I scroll down, the tracking function gets called". From some quick research, simulating scrolling in Enzyme is not easy (or even possible), which forces us to test things like this - which are effectively a test that we wired code up the way we thought we did. There's *value* in verifying that we wired code up the way we thought we did....but it'd so much more useful to test things from a user perspective - when I do X, our component tracks Y. 

I'm not requesting any changes, or making any comments or judgements on this code....I am just complaining about Enzyme 😭